azure_functions_worker/bindings/meta.py

```diff
@@ -25,11 +29,38 @@ def load_binding_registry() -> None:
     global BINDING_REGISTRY
     BINDING_REGISTRY = func.get_binding_registry()
 
-
-def get_binding(bind_name: str) -> object:
+    # The SDKs only support python 3.9+
+    if sys.version_info.minor > 8:
+        # Check if cx has imported sdk bindings library
+        try:
+            clients = importlib.util.find_spec('azure.functions.extension.base')
+        except ModuleNotFoundError:
+            # This will throw a ModuleNotFoundError in env reload because
+            # azure.functions.extension isn't loaded in
+            clients = None
+
+        # This will be None if the library is not imported
+        # If it is not None, we want to set and use the registry
+        if clients is not None:
+            import azure.functions.extension.base as clients
+            global SDK_BINDING_REGISTRY
+            SDK_BINDING_REGISTRY = clients.get_binding_registry()
+
+
+def get_binding(bind_name: str, pytype: typing.Optional[type] = None) -> object:
     binding = None
+
     registry = BINDING_REGISTRY
```

Comment:
nit: rename the registry to default_binding_registry or sort, and client_registry to sdk_binding_registry

Reply:
actually these two vars can be removed. we can use the uppercased global vars to cause less confusion.

Reply:
Good point, removed